### ISSUE TITLE
Stars before headings in org-mode

### ISSUE THREAD

**Issue by anupam312nwd** [ianpan870102/tron-legacy-emacs-theme#6]
Hi, Thanks for creating this awesome theme.
I am getting stars before headings in org-mode even though I have `org-hide-leading-stars` set to `t`. 
I am not getting it if I am using some other theme.

Can someone suggest how to remove these? Thanks!

![Screen Shot 2020-05-30 at 1 04 42 AM](https://user-images.githubusercontent.com/56342663/83320046-97043300-a211-11ea-80b9-048f2c6b0c8e.jpg)

Configuration: I am using Doom-Emacs. There are two ways of loading theme in Doom-Emacs: 
`(setq doom-theme #'tron-legacy)` and using `(load-theme #'tron-legacy)`. 
I am getting the same error by using any of these.

**Comment by ianpan870102** [comment]
Interesting, thanks for reporting. I assume you have tried restarting Emacs and also reverting the buffer and the issue persists?
May I also have your org-bullet config and emacs version?

**Comment by anupam312nwd** [comment]
> Interesting, thanks for reporting. I assume you have tried restarting Emacs and also reverting the buffer and the issue persists?
> May I also have your org-bullet config and emacs version?

Yes, I tried both.
`(after! org
 (setq
        org-ellipsis "↘" 
        org-superstar-headline-bullets-list '("❖" "✣" "■" "●" "◆" "▶" "◉" "◈" "○" "◇"))`
I think these are org-config related to bullets.
There are two other variables which might would have been relevant:
`org-superstar-remove-leading-stars` and `org-level-color-stars-only` with values `nil` or `t`.
I tried all combinations of these, but the issue still persists.

Emacs version: GNU Emacs 26.3.

**Comment by ianpan870102** [comment]
I see, it's the org-superstar package instead of org-bullets. I'll investigate further and hopefully get back with a solution

**Comment by ianpan870102** [comment]
Turns out it's not an uncommon problem, but the org-superstar devs have a **solution**: https://github.com/integral-dw/org-superstar-mode#hide-leading-stars
TL;DR, it works by setting the leading stars to whitespaces.

I just tried it and it **works** on my Emacs 27 like so:

![image](https://user-images.githubusercontent.com/36686900/83321450-52fd4680-a282-11ea-9f7c-d71c6c15fd45.png)

Here's my relevant config that might help you, even though I'm not using Doom Emacs:

```emacs-lisp
(use-package org-superstar
  :config
  (add-hook 'org-mode-hook (lambda () (org-superstar-mode 1)))
  (setq org-hide-leading-stars nil)
  (setq org-superstar-leading-bullet ?\s))
```

**Comment by anupam312nwd** [comment]
Hi, I tried these settings:
```elisp
(use-package org-superstar
  :config
  (add-hook 'org-mode-hook (lambda () (org-superstar-mode 1)))
  (setq org-hide-leading-stars nil)
  ;; (setq org-superstar-leading-bullet ?\s)
  (setq org-superstar-leading-bullet 32) ; tried both settings, was originally set to 32
  (setq org-superstar-headline-bullets-list '("❖" "✣" "■" "●" "◆" "▶" "◉" "◈" "○" "◇"))
  )
```
It's still giving the same issue. Maybe it's related to doom-emacs or emacs version 26.3.
I couldn't find an easy way to upgrade emacs.

**Comment by ianpan870102** [comment]
I noticed that you commented out `(setq org-superstar-leading-bullet ?\s))`, does it not make a difference whether you turn it on or off?

> Maybe it's related to doom-emacs or emacs version 26.3.

That's also possible, perhaps you can open an issue in doom-emacs' repo or org-superstar and reference this issue here.

> On a different note, the black stars before heading fonts turn white (for whatever reasons) in case I toggle between different themes without restarting emacs.

I also experience similar things with org-bullets when switching themes. I'm almost certain it has more to do with the face and font rendering of Emacs, instead of individual theme implementations.

> Actually, I can close this issue as it might not be directly related to this theme.

Sure, but before you do, can you try one last thing:
In `tron-legacy-theme.el`, try to set the org-hide face to be the same as background, namely this line:
``(org-hide       ((,class (:foreground ,bg1)))) ;; currently it's fg4`

**Comment by anupam312nwd** [comment]
> Sure, but before you do, can you try one last thing:
> In `tron-legacy-theme.el`, try to set the org-hide face to be the same as background, namely this line:
> `'(org-hide ((,class (:foreground ,bg1)))) ;; currently it's fg4`

I tried `bg1`. Actually, then I tried few different colors (like tron-yellow). It did not have any effect. But I removed box from around org-todo and bold it which took effect (which I wanted as gotten used to).

For someone reading later, I changed `org-code` and `org-verbatim` color as follows as (in your config) I found it not much different than the foreground:
```elisp
(custom-theme-set-faces
 'user
 '(org-code ((t (:foreground "#ff0800")))) ; Carmine red
 '(org-verbatim ((t (:foreground "#66ff00")))) ; lime green
 )
```
